src/comm.cpp

```diff
@@ -1410,6 +1410,9 @@ int make_prompt(struct descriptor_data * d)
               else
                 strcpy(str, "@v");
               break;
+            case 'V':       // Vision penalties in theory
+              sprintf(str, sizeof(str), "%d", calculate_vision_penalty(d->character));
```

Comment:
This should be an `snprintf` (sprintf is not memory-safe, and I'm working to remove it from the game where I see it). Other than that, yeah, this should work.